modules/comparison-of-solutions-to-provision-node-local-storage.adoc

```diff
@@ -0,0 +1,172 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent_storage/persistent_storage_local/overview-and-comparison-of-local-storage-options
```

Comment:
Since you're already going to make changes in this file, I'd update this entry to have the correct file name:
```suggestion
// * storage/persistent_storage/persistent_storage_local/ways-to-provision-local-storage.adoc
```

Reply:
Updated as per the suggestion.